Codemine/Extensions/String+Range.swift

```diff
@@ -42,9 +42,9 @@ public extension String {
      
      - returns: the range between the start of the first substring and the end of the last substring
      */
-    public func rangeFromString(_ string: String, toString: String, searchType: RangeSearchType = .leftToRight,  inRange: Range<Index>? = nil) -> Range<Index>? {
+    public func rangeFrom(string: String, toString: String, searchType: RangeSearchType = .leftToRight,  inRange: Range<Index>? = nil) -> Range<Index>? {
```

Comment:
I am pretty sure this one should be `range(from fromString: String, to toString: String, ....` :)